pallets/runtime/tests/src/settlement_test.rs

```diff
@@ -490,7 +490,6 @@ fn token_swap() {
         let instruction_details = Instruction {
             instruction_id,
             venue_id: venue_counter,
-            status: InstructionStatus::Pending,
             settlement_type: SettlementType::SettleOnAffirmation,
             created_at: Some(Timestamp::get()),
             trade_date: None,
```

Comment:
Should also check the instruction status at each call to `assert_instruction_details(instruction_id, instruction_details);`

Reply:
And at the two other spots like this below.